wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -3,7 +3,7 @@
     "pins": [
       {
         "package": "DifferenceKit",
-        "repositoryURL": "https://github.com/wireapp/DifferenceKit/",
+        "repositoryURL": "https://github.com/wireapp/DifferenceKit",
```

Comment:
this is annoying but I don't even know which one is correct?

Reply:
Are you working with Xcode 15.0.1?

Reply:
yes